starboard/tools/testing/test_runner.py

```diff
@@ -493,7 +493,13 @@ def MakeLauncher():
       # The filename is used by MH to deduce the target name.
       xml_filename = f"{target_name}_testoutput.xml"
       if out_path:
-        test_result_xml_path = os.path.join(out_path, xml_filename)
+        if self.platform in ["ps4", "ps5"]:
```

Comment:
Would be nice to just include all the windows platforms anyway, but maybe the xb1 platforms are being deploy from linux so not sure how applicable this request is. I think NX is fine here too, but we can skip win32 and xb1.